pkg/virt-controller/watch/preset_test.go

```diff
@@ -111,12 +111,12 @@ var _ = Describe("VM Initializer", func() {
 		})
 	})
 
-	Context("Conflict detection", func() {
+	Context("Override detection", func() {
```

Comment:
I don't see the following test cases.
* Verify VM spec after presets are applied
* Verify VM spec after presets are applied and VM spec overrides some values.

Essentially I'm talking about something that verifies that the resulting VM spec has been mutated in the way we'd expect after applying presets. 

Maybe that's occurring and I'm just overlooking the test case?

Reply:
The "Apply Presets" context tests specific traits of the VM post application. But you're right, there's no reason not to specifically check that [nothing was applied] in the Override and Conflict contexts as well.